recipes/pythonfmu/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pythonfmu" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e8789f32ca0422aac4d59735b5ad2189e6372816d73c4fd4e00878026163a0f7
+
+build:
+  number: 0
+  skip: True  # [py<36 or osx]
+  entry_points:
+    - pythonfmu-builder = pythonfmu.builder:main
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+    - pyfmi
+  commands:
+    - pytest --pyargs pythonfmu.tests
+  imports:
+    - pythonfmu
+
+about:
+  home: https://github.com/NTNU-IHB/PythonFMU
+  summary: A lightweight framework that enables the packaging of Python3.x code as co-simulation FMUs.
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  dev_url: https://github.com/NTNU-IHB/PythonFMU
+
+extra:
+  recipe-maintainers:
+    - fcollonval
+    - markaren
```

Comment:
@markaren are you OK with being a maintainer?

Reply:
yes